src/javaRosa.js

```diff
@@ -823,6 +823,8 @@ define([
             widgetClass = options.baseWidgetClass || widgets.richTextarea,
             widget, $input;
 
+        options.language = language;
```

Comment:
I think this is the closest use case: https://github.com/dimagi/Vellum/blob/master/src/javaRosa.js#L1152

I did it in this way so that it could be passed to the text and richText widgets, but I suppose it could have been done wherever this widget function was called